runtime/rococo/src/lib.rs

```diff
@@ -717,20 +717,6 @@ impl pallet_beefy_mmr::Config for Runtime {
 	type BeefyDataProvider = ParasProvider;
 }
 
-parameter_types! {
```

Comment:
Is it meant to be part of companion?

Reply:
Sort of yes. They are not being used anymore. I removed them while fixing some merge conflicts. I can bring them back, however creating an extra pr for this isn't it really worth :P

Reply:
No it's fine as long as it is not a mistake.